src/components/VideoPlayer/BaseVideoPlayer.tsx

```diff
@@ -145,6 +147,15 @@ function BaseVideoPlayer({
 
             if (event.fullscreenUpdate === VideoFullscreenUpdate.PLAYER_DID_DISMISS) {
                 isFullScreenRef.current = false;
+
+                currentVideoPlayerRef.current?.getStatusAsync?.().then((status) => {
+                    if (!('isMuted' in status)) {
+                        return;
+                    }
+
+                    updateVolume(status.isMuted ? 0 : status.volume || 1);
```

Comment:
here `|| 1` is necessary ?

Reply:
Instead, can we just set the volume like this: `updateVolume(status.isMuted ? 0: status.volume)`?

Reply:
No if we remove the `|| 1` we will have the [bug](https://github.com/Expensify/App/pull/39479#issuecomment-2037251863) mentioned above if we turn on the video sound in the `5th` step by pressing the volume icon.